java-manta-client-unshaded/src/main/java/com/joyent/manta/client/crypto/MantaEncryptedObjectInputStream.java

```diff
@@ -238,11 +238,11 @@ private long initializeCipher() {
         final int mode = Cipher.DECRYPT_MODE;
 
         try {
-            this.cipher.init(mode, secretKey, cipherDetails.getEncryptionParameterSpec(iv));
-
             if (startPosition != null && startPosition > 0) {
```

Comment:
Is the `null` check necessary for `startPosition` ?

Reply:
Similar comment for `L:243`

Reply:
Fixed